astroid/__pkginfo__.py

```diff
@@ -15,7 +21,12 @@
 
 extras_require = {}
 install_requires = ['lazy_object_proxy', 'six', 'wrapt']
-extras_require[':python_version<"3.4"'] = ['enum34', 'singledispatch']
+
+if py_version < (3, 4) and setuptools_version < '21.0.0':
```

Comment:
The acceptable setuptools version for `python_version` is 3.3 according to https://gsnedders.github.io/python-marker-test/results.html